### DISCUSSION TITLE
How do I get the Month as a number?

### DISCUSSION THREAD

**Discussion by Bas-Man** [time-rs/time#550] [Question] [answered]
Hi.
I have looked the documentation. I see that I can do `.year()` and get the year as as `2023` and `.month()` gets `February`. How do I get February as `2`?

**Comment by Bas-Man** [comment, marked as answer]
The answer is to use `as u8` because the Month enum has `#[repr(u8)]`

So one can simply do
```rust
use time::OffsetDateTime;

fn main() {
    let date = OffsetDateTime::now_utc().date();
    println!("{}", date);
    let year = date.month();
    let mon = date.month() as u8;
    println!("{} {}", year, mon);
    // try using the `time` crate here
}
```
[Playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=cd918c4e9f5f32197fbcf6222239ac77)

**Reply by jhpratt** [reply]
`From<Month>` is also implemented for `u8`, so you can do `u8::from(mon)` if you'd prefer. Either way works, and they have the same cost.
